Model/Events.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Klaviyo\Reclaim\Model;
+
+use Magento\Framework\Model\AbstractModel;
+
+class Events extends AbstractModel
+{
+    protected function _construct()
+    {
+        $this->_init('Klaviyo\Reclaim\Model\Resourcemodel\Events');
+        parent::_construct();
```

Comment:
Does this do anything? The parent class method is empty.

https://github.com/magento/magento2/blob/86dc050f2669617d1b34bc89a4a14522f008bf76/lib/internal/Magento/Framework/Model/AbstractModel.php#L204

Reply:
I think it's also called by the parent `__construct()` function which this class inherits: https://github.com/magento/magento2/blob/86dc050f2669617d1b34bc89a4a14522f008bf76/lib/internal/Magento/Framework/Model/AbstractModel.php#L173

Reply:
Yep, so my question is do we need to call the parent method here (and in the other model classes)? If it's left blank so child classes implement it what is this call to the parent method doing?

Reply:
yeah we probably don't need it, I just tested this, works without calling the parent construct, my bad